### DISCUSSION TITLE
Access to axis server

### DISCUSSION THREAD

**Discussion by IvanMeq** [AxisCommunications/discussions#15] [Axis Windows SDKs] [answered]
Hello.
Theres any way to connect an C# app with my Axis Server. Actualy I connect with AXIS Camera Station Client but I want to connect using this SDK. Is this possible?
Thanks in advance.

**Comment by marbali8** [comment, marked as answer]
Hello!
Thank you for your question. We don't support that anymore, but you can find other solutions in the [Axis Developer Community](https://www.axis.com/developer-community).
